src/transformStateWithClones.js

```diff
@@ -7,7 +7,29 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const resultArray = [state];
+  let removePropertiesClone = { ...state };
+
+  for (let i = 0; i < actions.length; i++) {
```

Comment:
you can use here `for of` loop
it will help you to avoid writing index every time